### DISCUSSION TITLE
Open issue for fixing the roles command

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#930] [First-Timers] [answered]
The `roles` command on [EddieBot](https://EddieHubCommunity/EddieBot) has an issue.
It would be great if someone can open an Issue for this :)
(and a PR would be awesome-!) 🟩  🤓 

## Screenshot:
![Screenshot 2021-02-17 at 9 06 05 AM](https://user-images.githubusercontent.com/62864373/108181137-8db74f80-712d-11eb-9c08-8475ccde2889.png)

The `Assigning this role to yourself:` message should only be added when the role is a self-assignable role.

## scope of error->
https://github.com/EddieHubCommunity/EddieBot/blob/a7bc337e793d485c5a50457bea21bc3ff33433e4/src/commandHandlers/roles.ts#L38-L44

## How to fix?:
Have a look at [roles.ts](https://github.com/EddieHubCommunity/EddieBot/blob/develop/src/commandHandlers/roles.ts) 
The `selfAssignableRoles` variable holds the self assignable roles.

**Comment by starlightknown** [comment, marked as answer]
opened an  [issue](https://github.com/EddieHubCommunity/EddieBot/issues/446)
